Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Oceanostics"
 uuid = "d0ccf422-c8fb-49b5-a76d-74acdde946ac"
 authors = ["tomchor <tomaschor@gmail.com>"]
-version = "0.11.4"
+version = "0.11.5"
```

Comment:
I'm not sure if it counts as breaking change the fact that one might have to install a new version of Julia to keep working. I guess there is no change in API so that means it's not breaking, right?

Reply:
Yeah I think technically this is non-breaking